conf.toml

```diff
@@ -1,11 +1,13 @@
+WebStaticFilesPath = "web"
+
 [AccountManager]
 Type = "unix"
 
 [InstanceManager]
 Type = "unix"
 
 [InstanceManager.GCP]
-ProjectId = ""
+ProjectID = ""
```

Comment:
This doesn't seem related to this change

Reply:
true, reverted.